TranslatorReasonerAPI.yaml

```diff
@@ -218,7 +218,9 @@ components:
       properties:
         results:
           description: >-
-            List of all returned Result objects for the query posed
+            List of all returned Result objects for the query posed.
+            The list SHOULD NOT be assumed to be ordered. The client SHOULD
+            reorder the results based on the 'score' property if it is present.
```

Comment:
Why do we need this last sentence? Do we want the client to do something other than not assuming the order matters?

Reply:
The second sentence is intended to make it clear where the client SHOULD look to find the ordering intended by the server.

Rather than telling the client what they should do, would it be preferable to replace the last sentence with:

The intended order of the results SHOULD be obtained by sorting the 'score' property if it is present.

Reply:
I'd prefer something even less imperative, like:

> The `score` property, if present, can be used to infer result rankings.

Reply:
okay, I agree that is better. I have updated the PR. How is the latest now?

Reply:
If the ARA does not rank (i.e. the list is randomly ordered), will this be explicitly declared, or it can just return the same score  value for all results, e.g. all zeros. Redundancy?

Reply:
The `score` property is not required by the schema. So therefore, if there is no `score` provided (or it is NULL, which is equivalent), then there is no ranking, since the list is declared in the documentation to be unordered,